external_reports/dropbear.yml

```diff
@@ -38,3 +38,15 @@ advisories:
       - https://matt.ucc.asn.au/dropbear/CHANGES
     reported: 2020-12-30
     severity: medium
+  - cve: CVE-2019-15599
```

Comment:
This doesn't look right. The CVE might be a typo for the wrong number.

I think it should be https://www.cvedetails.com/cve/CVE-2018-15599/

Reply:
Good catch. Thanks,